refinery/analysis_manager/tests.py

```diff
@@ -31,11 +34,12 @@ class AnalysisManagerTestBase(TestCase):
     def setUp(self):
         self.username = 'coffee_tester'
         self.password = 'coffeecoffee'
-        self.user = User.objects.create_user(self.username, '', self.password)
+        self.user_analysis = User.objects.create_user(self.username, '',
+                                                      self.password)
 
         analyses, self.dataset = make_analyses_with_single_dataset(
             1,
-            self.user
+            self.user_analysis
```

Comment:
For analysis monitoring, we now check for the user's data set perms. Below the analysis monitoring view uses both AnalysisManagerTestBase setup and ToolManagerBase setup which caused a conflict with scope for self.user.